app/forms/idv/api_image_upload_form.rb

```diff
@@ -224,6 +226,19 @@ def update_analytics(client_response)
           flow_path: params[:flow_path],
         ),
       )
+      pii_from_doc = client_response.pii_from_doc || {}
+      irs_attempts_api_tracker.idv_document_upload_submitted(
+        success: client_response.success?,
+        document_state: pii_from_doc[:state],
+        document_number: pii_from_doc[:state_id_number],
+        document_issued: pii_from_doc[:state_id_issued],
+        document_expiration: pii_from_doc[:state_id_expiration],
+        first_name: pii_from_doc[:first_name],
+        last_name: pii_from_doc[:last_name],
+        date_of_birth: pii_from_doc[:dob],
+        address: pii_from_doc[:address1],
```

Comment:
do we get an `address2` in the PII and if so, do we want to share that as well?

Reply:
i did not find any address line 2 key in get_results_response_success.json and we are not parsing in  PiiFromDoc as well